src/pages/index.js

```diff
@@ -14,41 +16,59 @@ const profileEditBtn = document.querySelector("#profile-edit-btn");
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileSubtextInput = document.querySelector("#profile-subtext-input");
 const profileModalForm = document.querySelector("#profile-modal-form");
-const profileTitle = document.querySelector("#profile-title");
-const profileSubtext = document.querySelector("#profile-subtext");
 
 const addBtn = document.querySelector("#add-button");
+const avatarEditBtn = document.querySelector("#avatar-edit-btn");
 
-const newCardTitleInput = document.querySelector("#new-card-title-input");
-const newCardLinkInput = document.querySelector("#new-card-link-input");
 const newCardModalForm = document.querySelector("#new-card-modal-form");
+const avatarModalForm = document.querySelector("#avatar-modal-form");
 
 const cardListElement = document.querySelector(".cards__list");
 
 /**
- * Functions
+ * Variables
  */
 
-// Section Functionality
+const editFormValidator = new FormValidator(config, profileModalForm);
+const cardFormValidator = new FormValidator(config, newCardModalForm);
+const avatarFormValidator = new FormValidator(config, avatarModalForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```